components/lib/biometric-prompt/src/main/java/mozilla/components/lib/auth/AuthenticationCallbacks.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.auth
+
+/**
+ * Callbacks for BiometricPrompt Authentication
+ */
+interface AuthenticationCallbacks {
```

Comment:
nit: naming. We use delegate in most places.

```suggestion
interface AuthenticationDelegate {
```

Reply:
Done